Pages/LanguageFeature.cs

```diff
@@ -0,0 +1,191 @@
+
+
+
+        public void AddLanguages(IWebDriver driver)
+        {
+            string[] languages = { "English", "French", "Hindi", "Gujarati" };
+            string[] langlevels = { "Fluent", "Basic", "Fluent", "Native/Bilingual" };
+
+            // Loop to add languages and levels
+            for (int i = 0; i < languages.Length; i++)
+            {
+                //In languages click on AddNew cell
+
+
+
+                addNewlanButton = driver.FindElement(addNewlanButtonLocator);
+                addNewlanButton.Click();
+
+                Thread.Sleep(3000);
+
+                // Find the language input element and enter the value
+
+
+
+                inputElement = driver.FindElement(inputElementLocator);
+                inputElement.SendKeys(languages[i]);
+
+
+                // Find the dropdown element and select the appropriate level
+
+                SelectElement levelDropdown = new SelectElement(driver.FindElement(By.XPath("//select[@class='ui dropdown'][@name=\"level\"]")));
+                levelDropdown.SelectByText(langlevels[i]);
+
+
+                Thread.Sleep(3000);
+
+                // Click the "Add" button
+
+
+                addlan = driver.FindElement(addlanLocator);
+                addlan.Click();
+                Thread.Sleep(3000);
+
+
+            }
+
+        }
+
+        public void VerifyLanguagesAdded(IWebDriver driver)
+        {
+            Thread.Sleep(3000);
+            // XPaths to locate language and level elements
+            string languageXPath = "//table[@class='ui fixed table']/tbody/tr/td[1]";
+            string levelXPath = "//table[@class='ui fixed table']/tbody/tr/td[2]";
+
+            // Find the elements containing languages and levels
+            var languageElements = driver.FindElements(By.XPath(languageXPath));
+            var levelElements = driver.FindElements(By.XPath(levelXPath));
+
+            // Store the languages and levels in lists
+            List<string> addedLanguages = new List<string>();
+            List<string> addedLangLevels = new List<string>();
+
+            // Extract text from each language element
+            foreach (var element in languageElements)
+            {
+                addedLanguages.Add(element.Text);
+            }
+
+            // Extract text from each level element
+            foreach (var element in levelElements)
+            {
+                addedLangLevels.Add(element.Text);
+            }
+
+            // Expected languages and levels for verification
+            string[] expectedLanguages = { "English", "French", "Hindi", "Gujarati" };
+            string[] expectedLangLevels = { "Fluent", "Basic", "Fluent", "Native/Bilingual" };
+
+            // Verify that the added languages match the expected languages
+            for (int i = 0; i < expectedLanguages.Length; i++)
+            {
+                Assert.AreEqual(expectedLanguages[i], addedLanguages[i], $"Language mismatch at index {i}");
```

Comment:
also are you asserting the message the pops up when you add the language or skill?